.github/renovate.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "prConcurrentLimit": 5,
+  "baseBranches": ["master","release-1.8","release-1.9","release-1.10"],
```

Comment:
It is not immediately obvious to me how we could automate this list, but if we have it manual for now we should add a step to the minor release issue template to update this after new releases 👍🏻